packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/wdio-services/VisualRegressionService.tool.mdx

```diff
@@ -95,12 +72,6 @@ the locale the test runner is testing against. This is used to categorize screen
 * **theme** `String` ( default: 'terra-default-theme' ) <br />
 the theme the test runner is testing against. This is used to categorize screenshots by theme.
 
-* **misMatchTolerance** `Number`  ( default: 0.01 ) <br />
```

Comment:
Seems like this would be something teams would use. Has our default mismatch tolerance proved to always be correct?

Reply:
We allow configuration from the test-level command do teams will still have the ability to define this, they just won't be able to define this at the wdio config level